### PR TITLE
support for rgb space delimited

### PR DESCRIPTION
**Pull request by NickyTope**
as per:
https://stylelint.io/user-guide/rules/list/color-function-notation/

### CODE REVIEW

**Comment by brenoprata10** [comment]
Thanks for your contribution, could you remove the re-alignment on the other rows?

**Comment by NickyTope** [comment]
My apologies, my opinionated formatter really went to work on the rest of those files, have reset them now.

File: lua/nvim-highlight-colors/colors.lua [hunk 1]
```diff
@@ -3,7 +3,7 @@ local table_utils  = require("nvim-highlight-colors.table_utils")
 
 local M = {}
 
-M.rgb_regex = "rgba?[(]+" .. string.rep("%s*%d+%s*", 3, ",") ..",?%s*%d*%.?%d*%s*[)]+"
+M.rgb_regex = "rgba?[(]+" .. string.rep("%s*%d+%s*", 3, "[, ]") .. "[, /]?%s*%d*%.?%d*%s*[)]+"
```
Comment (brenoprata10) [inline lua/nvim-highlight-colors/colors.lua:6]: I think white-space would be better visualized if we use %s instead.
```suggestion
M.rgb_regex = "rgba?[(]+" .. string.rep("%s*%d+%s*", 3, "[,%s]") .. "[, /]?%s*%d*%.?%d*%s*[)]+"
```

**Review by brenoprata10 — CHANGES_REQUESTED** [review]

**Comment by brenoprata10** [comment]
Merged! I am so happy to see that the community is helping so much :smiley:

**Pull request merged**